<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>OptPlugins</class>
 <widget class="QWidget" name="OptPlugins" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OptPluginsUI</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="TextLabel2" >
     <property name="text" >
      <string>Plugin Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cb_plugins" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Plugin Location:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_location" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>No plugin selected</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="cb_loadPlugin" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Load this plugin</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QWidget" name="pluginOptions" />
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
